Kuadrant/limitador-operator
# helm manifests template based

### What

Helm manifests do not contain a namespace object. It is all delegated to the helm tool. 

Discussion on https://github.com/Kuadrant/authorino-operator/issues/202

Additionally, added verify ci test to ensure chart manifests are up to date

### Verification steps

Preparation setup 
```
make kind-create-cluster
```

Install helm charts

```
make helm
bin/helm install limitador-operator ./charts/limitador-operator/ --namespace limitador-system --create-namespace
```

Check status

```
bin/helm status limitador-operator -n limitador-system
```
